manubot/cite/csl_item.py

```diff
@@ -152,6 +152,32 @@ def clean(self, prune: bool = True):
             self.validate_against_schema()
         return self
 
+    def set_date_variable(self, iso_date, variable="issued"):
```

Comment:
Should we look to the `note` as another option for how this setter should work?
```python
@note.setter
    def note(self, text: str):
```

Reply:
These setters only work for set functions that only take a single positional argument. Therefore, a single `@date.setter` would be unable to set `variable="issued"`. We could eventually add properties with setters like `date_issued`, but not a huge benefit for the extra code